apps/easypid/src/use-cases/ReceivePidUseCaseFlow.ts

```diff
@@ -45,8 +45,8 @@ export abstract class ReceivePidUseCaseFlow<ExtraOptions = {}> {
     return this.currentState
   }
 
-  protected errorCallbacks: AusweisAuthFlowOptions['onError'][] = [(e) => this.handleError(e)]
-  protected successCallbacks: AusweisAuthFlowOptions['onSuccess'][] = [
+  private errorCallbacks: AusweisAuthFlowOptions['onError'][] = [this.handleError]
+  private successCallbacks: AusweisAuthFlowOptions['onSuccess'][] = [
```

Comment:
I think we need to have these as protected, as well we need it as `(e) => this.handleError(e)` to prevent incorrect this binding